types/relay-runtime/index.d.ts

```diff
@@ -114,8 +115,30 @@ export interface MutationConfig<TOperation extends OperationType> {
 // ./network/RelayNetworkLoggerTransaction
 export interface RelayNetworkLog {
     label: string;
-    values: ReadonlyArray<unknown>;
+    values: ReadonlyArray<any>;
```

Comment:
Why is this change necessary?

Reply:
Just following the typing in source code of relay

https://github.com/facebook/relay/blob/master/packages/relay-runtime/network/RelayNetworkLoggerTransaction.js#L46

Reply:
Fair enough 👍